### PR TITLE
docs(tutorial): publish webpage interaction tutorial

### PR DESCRIPTION
**Pull request by ShihChun-H**
Because

- webpage pipeline tutorial draft completed

This commit

- webpage pipeline tutorial publication

### CODE REVIEW

**Review by GeorgeWilliamStrong — COMMENTED** [review]
**Minor formatting comment**
When I select the `formatted` output option for the [5 Seconds Previews pipeline](https://instill.tech/instill-wombat/pipelines/jumbotron-webpage-5-sec-previews) the output blocks alternate between having a grey shaded and white background. Is this intentional? To me it implies that the outputs follow some alternating structure when they are just a list. (See attached image).

CleanShot 2024-05-16 at 09 29 10@2x

File: tutorials/webpage-interaction.mdx
Comment (GeorgeWilliamStrong) [inline tutorials/webpage-interaction.mdx:1]: **Minor grammatical change**

Please change
Line 118: To extract content scraped from the webpage by Website Component

To
Line 118: To extract content scraped from the webpage by the Website Component

File: tutorials/webpage-interaction.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        - **API Key:** Reference the OpenAI API key previously set under **"Settings"** → **"Secrets"** as **`${secrets.openai-instill-api-key}`**.
+
+<ZoomableImg
+src="/tutorial-assets/webpage/T3.png"
+alt="Step 3 : Add Components"
+/>
+
+## 4. **Configure Output in Response Component**
+    - **General Output**:
+        - To extract content scraped from the webpage by Website Component, add **'Webpage Text Content'** field with Title **`Webpage Content`**, Key **`webpage_content`** and Value **`${web_scraper.output.pages[0].link_text}`**.
```
Comment (pinglin) [inline tutorials/webpage-interaction.mdx:118]:
```suggestion
        - To extract content scraped from the webpage by the Website component, add **'Webpage Text Content'** field with Title **`Webpage Content`**, Key **`webpage_content`** and Value **`${web_scraper.output.pages[0].link_text}`**.
```

**Review by pinglin — CHANGES_REQUESTED** [review]
Please change according to the review comments.

**Review by GeorgeWilliamStrong — APPROVED** [review]

**Pull request merged**